javascript/javascript2/week3/textlogout.js

```diff
@@ -0,0 +1,11 @@
+const buttton = document.getElementById('logtext');
+
+const logout = () => {
+	console.log('called after 2.5 seconds');
+};
+
+const Click = () => {
+	setTimeout(logout, 2500);
+};
+
+buttton.addEventListener('click', Click);
```

Comment:
I would suggest to pay attention to naming, especially when to use a verb or a noun,etc. See my suggestion bellow
```suggestion
buttton.addEventListener('click', OnClick);
```